build/templates/_library_interpreter.py.mako

```diff
@@ -78,7 +78,13 @@ class LibraryInterpreter(object):
         # Initialize _${config['session_handle_parameter_name']} to 0 for now.
         # Session will directly update it once the driver runtime init function has been called and
         # we have a valid session handle.
-        self._${config['session_handle_parameter_name']} = 0
+        self._set_session_handle()
+
+    def _set_session_handle(self, value=0):
```

Comment:
I'm not sure how pythonic it is to have these trivial private getter/setter methods.
We certainly don't use them throughout nimi-python. What prompted you to implement?

Reply:
In session.py's `close()` method, it was setting `self._interpreter._vi = 0`, but for gRPC it now needs to be an object. (This was failing one of NI-Digital's system tests for the `close()` method. I looked into adding an NI-FAKE test, but there's no good way to unit test this.) So I added `_set_session_handle()` with a default value, so that `close()` can just call `self._interpreter._set_session_handle()` and not have to copy the gRPC-specific default value to session.py.

I agree that it's not pythonic to call someone else's private methods, but it's also not pythonic to modify someone else's private members like we were doing before. I could make them public methods, if you want, since the class itself is already internal. All we'd have to worry about is possibly having to rename it later if some driver added a method with a conflicting name.

Reply:
Ok, so the only thing we get by having a method wrapping the property is the ability to have a default? I can buy that.
If the method is intended to be used outside of the (internal) class then I do think it should be "public".

Reply:
Ok, made them public.